client/src/components/friends-page/FriendsContent.css

```diff
@@ -56,7 +54,7 @@
 }
 
 .names-left {
-  font-family: "Cookie", cursive;
+  font-family: var(--cookie-text-title);
```

Comment:
Since you are mainly using this one from what i see, you could set this for all and then only define it again where it is different